inc/ui/namespace.php

```diff
@@ -109,7 +125,7 @@ class="<?php echo $group === $current_group ? 'current' : '' ?>"
  * @param Page $page
  * @param string $group
  */
-function render_page_subpages( Page $page, string $group ) {
+function render_page_subpages( Page $page, string $group, ?Page $current_page ) {
```

Comment:
I think you have to specify the default value as `null` but only if the param might not be passed in. Not sure if it's likely to happen.

Reply:
As it happens, I actually don't want a default here, and I think it's ok for it to be called with `null` is you _really_ need that.